### PR TITLE
fix pimd/langevin: improve usage of lj reduced units

### PR DESCRIPTION
**Pull request by Yi-FanLi**
**Summary**
This PR improves the usage of the *lj* units style in fix pimd/langevin. If one does not want to correspond the *lj* units to other physical units, this modification will automatically set the five parameters to 1. 

This PR does not break backward compatibility.

**Related Issue(s)**

**Author(s)**
Yifan Li
Princeton University
yifanl0716@gmail.com

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

**Implementation Notes**

**Post Submission Checklist**

- [x] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [ ] The source code follows the LAMMPS formatting guidelines
- [x] Suitable new documentation files and/or updates to the existing docs are included
- [ ] The added/updated documentation is integrated and tested with the documentation build system
- [ ] The feature has been verified to work with the conventional build system
- [ ] The feature has been verified to work with the CMake based build system
- [ ] Suitable tests have been added to the unittest tree.
- [ ] A package specific README file has been included or updated
- [ ] One or more example input decks are included

**Further Information, Files, and Links**

### CODE REVIEW

File: src/REPLICA/fix_pimd_langevin.h [hunk 1]
```diff
@@ -48,9 +48,9 @@ class FixPIMDLangevin : public Fix {
   double inverse_np;                       // 1.0/np
   double temp;                             // temperature
   double hbar;                             // Planck's constant
-  double lj_epsilon, lj_sigma, lj_mass;    // LJ unit energy, length, and mass scales
-  double other_planck;
-  double other_mvv2e;
+  double lj_epsilon = 1, lj_sigma = 1, lj_mass = 1;    // LJ unit energy, length, and mass scales
```
Comment (akohlmey) [inline src/REPLICA/fix_pimd_langevin.h]: The LAMMPS programming style requests to put initialization of class members into the constructor and not in the header.

**Review by akohlmey — CHANGES_REQUESTED** [review]
Please see https://docs.lammps.org/Modify_style.html#include-files-varied

File: src/REPLICA/fix_pimd_langevin.h (hunk 1 above)
Comment (Yi-FanLi) [reply src/REPLICA/fix_pimd_langevin.h]: Sorry my bad. I have just changed this.

**Comment by akohlmey** [comment]
@Yi-FanLi please merge with upstream to make certain there are no clashes and it tests cleanly. Thanks.

**Comment by Yi-FanLi** [comment]
> @Yi-FanLi please merge with upstream to make certain there are no clashes and it tests cleanly. Thanks.

Done. Thanks for your reminder. : )

**Review by akohlmey — APPROVED** [review]

**Review by sjplimp — APPROVED** [review]

**Pull request merged**